source/contracts/trading/ShareToken.sol

```diff
@@ -36,6 +36,14 @@ contract ShareToken is DelegationTarget, Extractable, ITyped, Initializable, Var
         return true;
     }
 
+    function trustedOrderTransfer(address _source, address _destination, uint256 _attotokens) public onlyCaller(bytes32("CreateOrder")) onlyInGoodTimes afterInitialized returns (bool) {
```

Comment:
The explicit `bytes32` conversion should not be necessary here.  Solidity will implicitly cast a string literal into a `bytes32` if the context requires it (in this case it does since the `onlyCaller` signature takes a `bytes32`.  What you can't do is assign a string literal to a `string` and then assign that `string` to a `bytes32`, but that caveat shouldn't apply here.